src/integration.tests/jest.config.js

```diff
@@ -0,0 +1,20 @@
+const { jest: jestConfig } = require('../../package.json');
+
+// eslint-disable-next-line prefer-object-spread/prefer-object-spread
+module.exports = Object.assign({}, jestConfig, {
+  rootDir: '../',
+  reporters: [
+    'default',
+    [
+      '../dist/reporter',
```

Comment:
This is fine to be `../dist/reporter` but it will break the circleCi pipeline as we don't do `npm run build` before `npm run test:integration`

Reply:
Can you update that?